module5/lessons/Week1/CoffeeChatKickoff.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Coffee Chat Kickoff
+---
+
+In Mod 4, you all did a great job with your first coffee chat! I know for some of you it was quite stressful, my first couple coffee chats definitely were! I hope you feel proud of yourself for breaking the ice and are now ready to keep this energy moving forward 🚀
+
+As we talked about in the CoffeeChatPD lesson there are many benefits of coffee chats including:

```

Comment:
I love that you included this review of the PD here in the lesson, rather than a link to the other session!